### PR TITLE
feat(*): add option to update required percent of tokens

### PR DESCRIPTION
**Pull request by DZariusz**
This feature is just for development purposes.
It makes developer life easier to have option to update `minimumStakingTokenPercentage`

### CODE REVIEW

**Comment by enddynayn** [comment]
👍 awesome thanks!

File: contracts/ChainConfig.sol [hunk 1]
```diff
@@ -60,6 +60,20 @@ contract ChainConfig is Ownable {
     return true;
   }
 
+  /// @dev this is just for development purposes! must be removed in real product implementation
+  function updateMinimumStakingTokenPercentage(uint8 _minimumStakingTokenPercentage)
```
Comment (therevoltingx) [inline contracts/ChainConfig.sol]: I think this should be passed in the constructor so you specify whether the chain does or doesn't support it.
Might be easier than launching two contracts with two versions of the code.

Comment (DZariusz) [reply contracts/ChainConfig.sol]: done

**Comment by enddynayn** [comment]
👍

**Pull request merged**